roles/engineblock/templates/parameters.yml.j2

```diff
@@ -1,4 +1,115 @@
 parameters:
+    domain: {{ base_domain }}
+    hostname: engine.{{ engine_base_domain }}
+    trusted_proxies:
+{% for engine_trusted_proxy_ip in engine_trusted_proxy_ips %}
+        - {{ engine_trusted_proxy_ip }}
+{% endfor %}
+    encryption_keys:
+{% for key, value in engine_keys.items() %}
+        {{ key }}:
+            publicFile: {{ value.publicFile }}
+            privateFile: {{ value.privateFile }}
+{% endfor %}
+    forbidden_signature_methods: {  }
+    allowed_acs_location_schemes:
+        - http
+        - https
+    php_settings:
+        memory_limit: 128M
+        display_errors: '{{ php_display_errors }}'
+        error_reporting: '6135'
+        date.timezone: {{ timezone }}
+        sendmail_from: 'OpenConext EngineBlock <openconext-engineblock@openconext.org>'
+    api.users.metadataPush.username: {{ engine_api_metadata_push_user }}
+    api.users.metadataPush.password: {{ engine_api_metadata_push_password | replace("%","%%") }}
```

Comment:
Why is this replace necessary? Can't you just quote the parameters (like sendmail_from above)?

Reply:
The problem is that Symfony is using '%' to replace variables. Therefore it needs to get escaped else the character will be removed. This logic was implemented in the dumper script and it was moved here to have the same behaviour.

Reply:
This is behaviour is of Symfony itself and is on top of the default Yaml behaviour so it wouldn't be suffiecient to just escape Yaml.

Reply:
:+1: